_projects/2023/100481433.Rmd

```diff
@@ -0,0 +1,279 @@
+
+### **Library and the data set with the frequency variables**
+
+```{r}
+data <- read.csv("C:/Users/sofia/OneDrive/Documentos/Master Computational Social Sciences/Data Visualization/Graph/Data World csv.csv", 
+                 sep = ";", 
+                 dec = ",")
+library(ggplot2)
+library(extrafont)
+library(dplyr)
+library(tidyr)
+```
+
+## **Building the chart**
+
+### **Data**
+
+The data scoured from Our World in Data needed to be converted to frequency tables . The graph's vertical axis, which I aimed to replicate, utilized cumulative frequency as a metric, necessitating the data's conversion. Subsequently, I identified and chose the specific columns needed for the replication, resulting in the creation of a new data-set named *df_reduced*.
+
+```{r}
+df_reduced <- data %>%
+  select(Year, Frec.Non.electoral.autocracies, Frec.One.party.autocracies, Frec.Multi.party.autocracies.without.elected.executive,
+         Frec.Multi.party.autocracy, Frec.Exclusive.democracies, Frec.Male.democracies, Frec.Electoral.democracies, Frec.Polyarchies) 
+
+#FREC - refers to the frecuency of the
+
+
+custom_order <- c("Frec.Non.electoral.autocracies", "Frec.One.party.autocracies", "Frec.Multi.party.autocracies.without.elected.executive",
+                  "Frec.Multi.party.autocracy", "Frec.Exclusive.democracies", "Frec.Male.democracies", "Frec.Electoral.democracies", "Frec.Polyarchies")
+```
+
+To enhance accessibility and facilitate the chart-building process, I converted the new dataset into tidy data. During this conversion, several changes were implemented, and intervals were established for the axis.
+
+```{r}
+df_long <- df_reduced %>%
+  pivot_longer(cols = -Year, names_to = "Democracy_Type", values_to = "n")
+
+df_long <- df_long %>%
+  mutate(Democracy_Type = factor(Democracy_Type, levels = custom_order))
+
+intervalos_x <- c(1789, 1850, 1900, 1950, 2000, 2022)
+
+intervalos_y <- seq(0, 100, by = 0.2)  
+
+```
+
+Due to each variable having an associated color. The following color vector was created for each value.
+
+```{r}
+colores_democracias <- c( "Frec.Non.electoral.autocracies" = "#d73027", "Frec.One.party.autocracies" = "#f46d43", "Frec.Multi.party.autocracies.without.elected.executive" = "#fdae61", "Frec.Multi.party.autocracy" = "#fee090", "Frec.Exclusive.democracies" = "#e0f3f8", "Frec.Male.democracies" = "#abd9e9", "Frec.Electoral.democracies" = "#74add1", "Frec.Polyarchies" = "#4575b4" )
+```
+
+### Starting with the graphic
+
+To construct the visual representation, I utilized the *geom_area* function from *ggplot*, tailored for generating **Stacked Area Charts.**
+
+```{r}
+plot <- ggplot(df_long, aes(x = Year, y = n, fill = Democracy_Type)) +
```

Comment:
You can use `fig.align='center'` as an option in your chunks to get centered figures.